api/test/application/user/user-register-post.js

```diff
@@ -1,16 +1,17 @@
-const Lab = require('@hapi/lab');
-const Code = require('@hapi/code');
-const getServer = require('../../server');
-const { truncateAll } = require('../../helpers/db');
-const { describe, it, before, beforeEach, after, afterEach } = exports.lab = Lab.script();
-const { mockSendEmail } = require('../../helpers/mocks');
-const sinon = require('sinon');
-const factory = require('../../helpers/factory');
+const Lab = require("@hapi/lab");
+const Code = require("@hapi/code");
+const getServer = require("../../server");
+const { truncateAll } = require("../../helpers/db");
+const { describe, it, before, beforeEach, after, afterEach } = (exports.lab =
+  Lab.script());
+const { mockSendEmail } = require("../../helpers/mocks");
+const sinon = require("sinon");
+const factory = require("../../helpers/factory");
 const { expect } = Code;
-const uuid = require('uuid');
-const bookshelf = require('../../../src/lib/bookshelf');
-const querystring = require('querystring');
-const webhookService = require('../../../src/application/webhook/webhook-service');
+const uuid = require("uuid");
+const bookshelf = require("../../../src/lib/bookshelf");
+const querystring = require("querystring");
+const webhookService = require("../../../src/application/webhook/webhook-service");
```

Comment:
Just formatting, no imports were changed

Reply:
Is this a result of prettier running in your editor? If so can you turn that off in your editor for just this project OR just send a PR that prettiers the whole thing and adds prettier as a CI check?